services/traction/api/services/innkeeper.py

```diff
@@ -67,3 +70,21 @@ async def create_new_tenant(
         else:
             # what to return or throw here?
             return
+
+
+async def hard_delete_tenant(tenant: TenantRead, db: AsyncSession):
+    q = select(Tenant).where(Tenant.id == tenant.id)
+    result = await db.execute(q)
+    db_tenant = result.scalar_one()
+    if not tenant:
+        raise Exception
+
+    response = multitenancy_api.multitenancy_wallet_wallet_id_remove_post(
```

Comment:
this ends up using the tenant's wallet token correct? so it's a little weird to do the acapy work for the tenant in the innkeeper? could do the acapy work in tenant service, then call innkeeper to clean up the traction tables only.

Reply:
I'm not sure how/why, but i did not end up needing the tenant to provide the acapy wallet_key. so the innkeeper endpoint with tenant_id is all you need

Reply:
but i believe that since you are calling it via a tenant endpoint, the tenant's token gets put into the context which is what those acapy api clients use... so this is using the current context which is the tenant which is why it works.  so the innkeeper can't really call this function...

Reply:
hmmmm, i'm calling from the innkeeper with the innkeeper auth token... so i'm not sure how acapy is using it's endpoint.

Reply:
sorry, my bad, yes, innkeeper route... i see. confused myself i guess...
so, inkeeper route, innkeeper service, innkeeper token, all good.